src/tlo/methods/tb.py

```diff
@@ -2023,6 +2035,7 @@ def __init__(self, module, person_id, suppress_footprint=False):
         self.TREATMENT_ID = "Tb_Test_Xray"
         self.EXPECTED_APPT_FOOTPRINT = self.make_appt_footprint({"DiagRadio": 1})
         self.ACCEPTED_FACILITY_LEVEL = '1b'
+        self.EQUIPMENT = {'X-ray machine', 'X-ray viewer'}  # TODO: make an x-ray pkg with these items
```

Comment:
In this again the case when if the person is already dead the equipment will be already set to {'X-ray machine', 'X-ray viewer'} so it should be overwritten to empty set in such case?

(The same in `HSI_Tb_Xray_level2`)

Reply:
I've set this now to a blank equipment list unless person meets conditions for xray

Reply:
Or on the other hand, I can update the other ones to have them set to empty set and only if person alive and consumables available, equipment will be set.

Reply:
Happy to go with your preference and match what others have also done.

Reply:
Okay, I will do the decision and change what is needed and merge this. So I guess your part is done! Thank you again, it was pleasure to work with you. You are so efficient.

Reply:
Thank you for all your guidance and sorry for the all back-and-forth :-)